Classes/ActorTraits/Rest.php

```diff
@@ -41,13 +41,19 @@ public function iDoARequestOnWithParameters(string $requestType, string $url, Ta
             throw new \Exception('Request type "' . $requestType . '" not yet implemented', 1693489230);
         }
 
-        $parameterArray = [];
+        $files = [];
         foreach ($parameters->getRows() as $index => $row) {
-            $row[1] = $this->convertStringToValue($row[1]);
-            $parameterArray[$row[0]] = $row[1];
+            if (strncmp($row[0], '$_FILES.', strlen('$_FILES.')) === 0) {
```

Comment:
magst du das noch im Kommentar über der Methode kurz erwähnen, dass man mit $_FILES... Dateien hinzufügen kann in der Tabelle?